### PR TITLE
adding duathread armour

### PR DESCRIPTION
**Pull request by a-person5660**
## About the PR
added durathread armour

## Why / Balance
contributes towards the completion of  #304

## Media
https://github.com/Goob-Station/Goob-Station/assets/40855173/87b83194-2c3d-416f-bf9b-74706938ce04

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
🆑 
- add: durathread armour, web vest and jumpsuit is now craftable in the clothes printer when emagged

### CODE REVIEW

**Comment by a-person5660** [comment]
this should be working, it is building on my end. im not sure whats causing it.

**Comment by a-person5660** [comment]
it should be good to merge now

File: Resources/Prototypes/Recipes/Lathes/clothing.yml [hunk 1]
```diff
@@ -1306,3 +1306,29 @@
   completetime: 1
   materials:
     Cloth: 100
+
+- type: latheRecipe # Goobstation - adding durathread armoured clothing
```
Comment (Aidenkrz) [inline Resources/Prototypes/Recipes/Lathes/clothing.yml:1310]: this should be in its own file in /goobstation/

**Review by Aidenkrz — CHANGES_REQUESTED** [review]
Move the latherecipes into goobstation, then this should be good 👍

**Comment by a-person5660** [comment]
now it should be good to merge

**Pull request merged**